Assets/Scripts/Functional Definitions/Abilities/AbilityUtilities.cs

```diff
@@ -1,49 +1,49 @@
-﻿using System.Collections;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 using System.Reflection;
 
 public enum AbilityID
 {
-	Unused,
-	SpeedThrust,
-	ShellBoost,
-	MainBullet,
-	Beam,
-	Bullet,
-	Cannon,
-	Missile,
-	Torpedo,
-	Laser,
-	SpawnDrone,
-	CoreHeal,
-	Energy,
-	Speed,
-	SiegeBullet,
-	SpeederBullet,
-	Harvester,
-	ShellRegen,
-	ShellMax,
-	EnergyRegen,
-	EnergyMax,
-	Command,
-	CoreRegen,
-	CoreMax,
-	Stealth,
-	DamageBoost,
-	AreaRestore,
-	PinDown,
-	Retreat,
-	Absorb,
-	ActiveShellRegen,
-	ActiveCoreRegen,
-	ActiveEnergyRegen,
-	Disrupt,
-	Control,
-	InvertTractor,
-	Bomb,
-	Ion
+	Unused, //1
```

Comment:
This is unnecessary and also wrong since Unused is ID 0

Reply:
whoops, forgot to remove that; that was a (clearly incorrect) comment I left on my own master, I didn't intend it to get pushed :P